### ISSUE TITLE
Links to wasix are dead

### ISSUE THREAD

**Issue by mattbishop** [wapm-packages/curl#1]
Did the wasix project move? https://github.com/wasmer/wasix is gone or private.

**Comment by zlonqi** [comment]
> Did the wasix project move? https://github.com/wasmer/wasix is gone or private.

Hi, Have you successfully compiled libcurl? I ran into the same problem: I would like to compile libcurl to wasm library but don't know how to do it.

**Comment by turbolent** [comment]
@syrusakbary Do you know where the WASIX dependency went? 1b65298c5595902fb385727dd1f7f95c6edbebe9 adds instructions on how to compile this, but as pointed out, the WASIX repo is gone/private.

I looked at the imports of the `curl.wasm` file: Is there any documentation for the imports, like `wasio_unstable.socket_create`? I cannot find anything in the wasmer project that implements it

**Comment by StEvUgnIn** [comment]
`wasix` was replaced with `wasienv`
https://github.com/wasienv/wasienv

How to install it:
https://docs.wasmer.io/ecosystem/wasienv/getting-started

**Comment by syrusakbary** [comment]
> wasix was replaced with wasienv

Not really, we just decided to iterate on it privately to make sure it was in good shape before publicly releasing it!
Stay tuned for new announcements :)
